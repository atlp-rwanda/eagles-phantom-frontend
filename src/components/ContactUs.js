import React,{ Component } from 'react';

const ContactUs=()=>{
  return <h1>Contact Us</h1>
}

export default ContactUs;