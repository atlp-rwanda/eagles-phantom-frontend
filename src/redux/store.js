import {createStore, applyMiddleware } from 'redux';
import { composeWithDevTools} from 'redux-devtools-extension';
import thunk from 'redux-thunk';
import promise from 'redux-promise-middleware';
import allReducers from './reducers/';

// const rootReducer = combineReducers({
//   count: counterReducer,
// });

// const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;

const middleware = applyMiddleware(thunk, promise)
const store = createStore(allReducers, {}, composeWithDevTools(middleware));

export default store;
