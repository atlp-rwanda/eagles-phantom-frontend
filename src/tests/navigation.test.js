import React, { Component } from 'react';
import { Provider } from "react-redux";
import store from "../redux/store.js";
import Error from '../components/Error';
import Login from '../components/Login';
import { render } from '@testing-library/react';
import AboutUs from '../components/AboutUs.js';
import ContactUs from '../components/ContactUs';


const error = () => {
    return render(
      <Error />
    )
  };
  describe("Error",()=>{
    it("should render the component",()=>{
        const { asFragment }=error();
        expect(asFragment(<Error />)).toMatchSnapshot()
    });
});
const login=()=>{
    return render(
        <Login />
    )
};
describe("Login",()=>{
    it("should render the component",()=>{
        const { asFragment }=login();
        expect(asFragment(<Login />)).toMatchSnapshot()
    });
});
const about=()=>{
    return  render(
        <AboutUs />
    )
};
describe("AboutIs",()=>{
    it("should render the component",()=>{
        const { asFragment }=about();
        expect(asFragment(<AboutUs />)).toMatchSnapshot()
    })
});
const contact=()=>{
    return render(
        <ContactUs />
    )
};
describe("ContactUs",()=>{
    it("should render the Component",()=>{
        const { asFragment }=contact();
        expect(asFragment(<ContactUs />)).toMatchSnapshot()
    })
});

