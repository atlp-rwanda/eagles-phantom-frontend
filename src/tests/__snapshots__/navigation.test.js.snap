// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutIs should render the component 1`] = `
<DocumentFragment>
  <div>
    <h1
      class="title is-1"
    >
      This is the About Page
    </h1>
    <p>
      Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Duis consequat nulla ac ex consequat, in efficitur arcu congue. Nam fermentum commodo egestas.
    </p>
  </div>
</DocumentFragment>
`;

exports[`ContactUs should render the Component 1`] = `
<DocumentFragment>
  <h1>
    Contact Us
  </h1>
</DocumentFragment>
`;

exports[`Error should render the component 1`] = `
<DocumentFragment>
  <h1>
    Page not found!
  </h1>
</DocumentFragment>
`;

exports[`Login should render the component 1`] = `
<DocumentFragment>
  <h1>
    Login
  </h1>
</DocumentFragment>
`;
